x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -227,6 +227,15 @@ export function AlertDetails() {
     */
     isAlertDetailsEnabledPerApp(alertDetail.formatted, config) ? (
       <>
+        <EuiSpacer size="m" />
+        <StaleAlert
+          alert={alertDetail}
```

Comment:
could we pass `alertDetail.formatted` instead of `alertDetail`?

Reply:
Sure, what is the difference?

Reply:
I mean, inside the `StaleAlert` component I'm using `alert?.formatted` all the time

Reply:
right, I was thinking if we pass here `alertDetail.formatted` directly, then we don't need to use `.formatted` everywhere in `StaleAlert` component and we are not using anything else from `alertDetail` in `StaleAlert`.

Reply:
Done